spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -312,10 +301,7 @@ function startFlushUpgradeHandshake(
   // either timeout height or timestamp must be non-zero
```

Comment:
In the comment al line 287, we should remove "and the las packet sent".

Reply:
Yes, good catch!